apps/buildog/src/components/themes/cyan-header.tsx

```diff
@@ -0,0 +1,39 @@
+import Link from "next/link";
+
+export default function CyanHeader() {
+  return (
+    <header className="bg-cyan-900 border-b border-gray-300">
+      <div className="container mx-auto px-4 py-4">
+        <div className="flex flex-col sm:flex-row items-center justify-between space-y-4 sm:space-y-0">
+          <Link href="/" className="text-2xl font-bold text-white transition-colors">
+            Buildog
+          </Link>
+          <nav>
+            <ul className="flex space-x-6">
+              <li>
+                <Link href="/" className="text-white transition-colors">
+                  Welcome
+                </Link>
+              </li>
+              <li>
+                <Link href="/#" className="text-white transition-colors">
+                  To
+                </Link>
+              </li>
+              <li>
+                <Link href="/#" className="text-white transition-colors">
+                  Buildog
+                </Link>
+              </li>
+              <li>
+                <Link href="/#" className="text-white transition-colors">
+                  App
+                </Link>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
```

Comment:
This suggestion is a new design. Can you change it please? Also can you add Icons? you can using radix icons for now. And can you please change the folder name and component name with name-center-header.tsx?

```suggestion
     <header className="bg-background px-4 py-3 shadow-sm sm:px-6 md:py-4">
    <div className="container mx-auto flex items-center justify-between">
      <div className="flex items-center space-x-4">
        <Link href="#" className="text-muted-foreground hover:text-foreground" prefetch={false}>
          {/* <TwitterIcon className="h-5 w-5" /> */}
          <span className="sr-only">Twitter</span>
        </Link>
        <Link href="#" className="text-muted-foreground hover:text-foreground" prefetch={false}>
          {/* <FacebookIcon className="h-5 w-5" /> */}
          <span className="sr-only">Facebook</span>
        </Link>
        <Link href="#" className="text-muted-foreground hover:text-foreground" prefetch={false}>
          {/* <InstagramIcon className="h-5 w-5" /> */}
          <span className="sr-only">Instagram</span>
        </Link>
      </div>
      <Link href="#" className="text-2xl font-bold tracking-tight" prefetch={false}>
        The Blog
      </Link>
      <nav className="hidden space-x-4 md:flex">
        <Link href="#" className="text-muted-foreground hover:text-foreground" prefetch={false}>
          Blog
        </Link>
        <Link href="#" className="text-muted-foreground hover:text-foreground" prefetch={false}>
          About
        </Link>
        <Link href="#" className="text-muted-foreground hover:text-foreground" prefetch={false}>
          Contact
        </Link>
      </nav>
    </div>
  </header>
```